package.json

```diff
@@ -8,7 +8,7 @@
     "url": "https://github.com/webpack/webpack-cli.git"
   },
   "engines": {
-    "node": ">=10.13.0"
+    "node": ">=14.0.0"
```

Comment:
Let's make it 14.15.0 first LTS of node v14. This is what we are doing in all loaders and plugins.

Reply:
Yeah, sounds good.

Reply:
Done